src/components/MapMenu.tsx

```diff
@@ -16,69 +16,78 @@ interface MapMenuProps {
 export default function MapMenu(props: MapMenuProps) {
   return (
     <div className='h-100'>
-      <h2 className='my-3 d-none'>Layers</h2>
+      <div className="d-flex flex-row flex-wrap justify-content-center">
```

Comment:
The `.h-100` class is not longer needed with the new layout, so all these flex classes can just move up to that div and the extra wrapping div removed.